core/templates/dev/head/editor/RouterServices.js

```diff
@@ -118,6 +124,18 @@ oppia.factory('routerService', [
       }
     };
 
+    var _actuallyNavigate = function(pathType, newStateName) {
+      if (pathType != 'preview' && pathType != 'gui') {
```

Comment:
Oh, sorry, you're right. My bad.

Let's define two new constants: SLUG_GUI and SLUG_PREVIEW, and use those instead. A slug is the 'end part' of a URL. The existing constants are, I believe, already used for other things, and you don't want to just change them because this may break other parts of the app.